packages/host-inventory/api.ts

```diff
@@ -2885,7 +2922,7 @@ export interface YumRepo {
 export const AccountsStalenessApiAxiosParamCreator = function (configuration?: Configuration) {
     return {
         /**
-         * Create account staleness record. Required permissions: inventory:TODO:write
+         * Create account staleness record. Required permissions: staleness:staleness:write
```

Comment:
hm. what happened here? It seems that only the docs changed, but I wonder where it gets these permissions from.

Reply:
hmm, I have no context on this. The generation works automatically. I will investigate why.

Reply:
BTW, this should be due to the change in the OpenAPI spec by the backend team

Reply:
Yes, it is due to an update in the API spec: https://developers.redhat.com/api-catalog/api/inventory

Reply:
Oh, yeah! Thank you for investigating this! Should we maybe point the backend team at this. It doesn't seem correct, or?

Reply:
Yup, reported